perfkitbenchmarker/providers/profitbricks/flags.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from perfkitbenchmarker import flags
+
+
+# Locations
+US_LAS = 'us/las'
+DE_FKB = 'de/fkb'
+DE_FRA = 'de/fra'
+
+# Zones
+ZONE_1 = 'ZONE_1'
+ZONE_2 = 'ZONE_2'
+
+flags.DEFINE_string('profitbricks_config',
+                    os.getenv('PROFITBRICKS_CONFIG',
+                              '~/.config/profitbricks-auth.cfg'),
+                    ('Path to config file containing your email and password. '
+                     'Can also be set via $PROFITBRICKS_CONFIG environment '
+                     "variable.\n(File format: email:password)"))
+
+flags.DEFINE_enum('location',
```

Comment:
It is expected - "zones" is the flag name because it can take a list of zones - it's pretty easy to mess up if you use it this way though, so I usually recommend that users stick to passing it a single zone or using configs if they need to run with more than one zone. It used to be the case that gflags would accept incomplete flag names as long as they could be unambiguously resolved to known flag names (so using zone instead of zones would work). This might still be the case if you set an option somewhere - I'm not sure.
 
If profitbricks users are used to the two parameter system, then we should probably keep it that way. I'd recommend defining a  ProfitBricksVmSpec so that users can use configs to specify the location (and ram+cores). This would let users do things like running benchmarks with different machine types for servers and clients.